datasets/nasa-nex-gddp-cmip6/collection/collection.json

```diff
@@ -0,0 +1,757 @@
+{
+  "type": "Collection",
+  "id": "nasa-nex-gddp-cmip6",
+  "stac_version": "1.0.0",
+  "description": "{{ collection.description }}",
+  "links": [
+    {
+      "rel": "root",
+      "href": "/collection.json",
+      "type": "application/json",
+      "title": "Earth Exchange Global Daily Downscaled Projections (NEX-GDDP-CMIP6)"
+    },
```

Comment:
I think this link can / should be removed.

Reply:
Removed.